### DISCUSSION TITLE
The problem is in the creation of a farming pool, I was deducted 10 xch from my balance.

### DISCUSSION THREAD

**Discussion by nailhb** [Chia-Network/chia-blockchain#12565] [General]
Hello! The problem is in the creation of a farming pool, I was deducted 10 xch from my balance. All photo evidence is there. Please return 10 xch to my balance.

**Comment by nailhb** [comment]
![Снимок](https://user-images.githubusercontent.com/109899000/180644640-bd411c3c-29ed-47a2-9ed2-3135090371bc.PNG)
![xch](https://user-images.githubusercontent.com/109899000/180644652-88752ee7-94f2-4e60-8542-3deb1bba0745.PNG)

**Comment by loppefaaret** [comment]
lets stick to #12572
